tests/test_collection.py

```diff
@@ -0,0 +1,54 @@
+from .test_base import TestBase
+from django.urls import reverse
+
+
+class CollectionTest(TestBase):
+    def test_logged_in_user_can_post_collection(self):
+        # log in as owner
+        self.client.login(username="owner", password="password123")
+        # try to post a collection
+        response = self.client.post(reverse('collection-list'),
+                                    {'title': 't', 'description': 'd', 'source': 'mst3k', 'tags': ''})
+        # assert success
+        self.assertEqual(response.status_code, 201)
+
+    def test_user_can_edit_own_collection(self):
+        # log in as owner
+        self.client.login(username="owner", password="password123")
+        # attempt to patch collection
+        response = self.client.patch(reverse('collection-detail', args=[self.collection.id]),
```

Comment:
You could add `self.collection_url = reverse('collection-detail', args=[self.collection.id]` since it's used in so many places!